x-pack/test/alerting_api_integration/spaces_only/tests/alerting/scheduled_task_id.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { getUrlPrefix, TaskManagerDoc } from '../../../common/lib';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function createScheduledTaskIdTests({ getService }: FtrProviderContext) {
```

Comment:
Should we add some more test cases here?

1. What happens when a task already exists with an ID different than the rule ID?
2. What happens when a task already exists with an ID identical to *a different* rule? (do we consider this possible, UUIDs and all?)